config/application.yml.default

```diff
@@ -17,6 +17,7 @@
 aamva_auth_request_timeout: 5.0
 aamva_auth_url: 'https://example.org:12345/auth/url'
 aamva_cert_enabled: true
+aamva_state_daily_maintenance_windows: '{"DC": {"start_time": "00:00", "end_time": "06:00", "tz": "America/New_York"}, "KY": {"start_time": "02:50", "end_time": "06:40", "tz": "America/New_York"}}'
```

Comment:
This is:

1. Incomplete; we know of many more instances,
2. Defined in a non-public AAMVA document I don't have access to, and
3. Kinda clunky to store on one line here if we're going to add a half-dozen other states.

I wanted to express them in local time because that's how I saw them mentioned, rather than dealing with UTC shifts at DST. Incidentally, all the examples I saw were in "ET".

Some maintenance windows are not daily; there are a few states that have Sunday windows and at least one with a Tuesday window.

Reply:
I actually thought about doing something like that. I didn't want to write a cron parser, though it looks like we're already using [fugit](https://github.com/floraison/fugit) so maybe I don't have to. 🤔